coremltools/converters/tensorflow/_tf_converter.py

```diff
@@ -208,18 +208,28 @@ def _graph_def_from_saved_model_or_keras_model(filename):
     :return: TensorFlow GraphDef object.
     """
     try:
-        import tensorflow as tf
-        from tensorflow.python.keras.saving import saving_utils as _saving_utils
-        from tensorflow.python.framework import convert_to_constants as _convert_to_constants
-        model = tf.keras.models.load_model(filename)
-        tf.keras.backend.set_learning_phase(False)
-        func = _saving_utils.trace_model_call(model)
-        concrete_func = func.get_concrete_function()
-        # concrete_func = model.signatures[tf.saved_model.DEFAULT_SERVING_SIGNATURE_DEF_KEY]
-        frozen_func = _convert_to_constants.convert_variables_to_constants_v2(concrete_func)
-        graph_def = frozen_func.graph.as_graph_def(add_shapes=True)
+       import tensorflow as tf
```

Comment:
I'm think I'm seeing 3 spaces vs. 4 spaces which are still inconsistent (is that GitHub display issue?). But it should not block this change.

Reply:
you are right. I have fixed it now :).